K8s-deployment/Charts/fast-api-server/example-aws-resource-values.yaml

```diff
@@ -0,0 +1,24 @@
+ingress:
+  hostname: fas-hostname 
+  annotations: {
+    cert-manager.io/cluster-issuer: "letsencrypt-staging" # use letsencrypt-prod if proper letsencrypt certs are required
+  }
+  
+## fastApiServer resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param fastApiServer.resources.limits The resources limits for the fastApiServer containers
+## @param fastApiServer.resources.requests The requested resources for the fastApiServer containers
+##
+fastApiServer:
+  nodeSelector:
+    node.kubernetes.io/instance-type: t3a.medium
+  resources:
```

Comment:
Update resources based on what Mayur has shared, keep those as requests, and limits can be +50% of that.